near-workspaces/__tests__/market.ava.ts

```diff
@@ -77,18 +77,14 @@ workspace.test(
         //     })
         // })
         // log(`✔  John failed to bid using alice account\n`)
-
-        /**
-         *  @todo fix in contract
-         *  https://github.com/curaOS/contracts/issues/94
-         * */
-        // await test.throwsAsync(async () => {
-        //     await call_set_bid(contract, john, {
-        //         tokenId: minted.id,
-        //         bid: {... john_example_bid, amount: NEAR.parse("0.05N") }
-        //     })
-        // })
-        // log(`✔  John failed to bid less than `min_bid_amount`\n`)
+        
+        await test.throwsAsync(async () => {
+            await call_set_bid(contract, john, {
+                tokenId: minted.id,
+                bid: {... john_example_bid, amount: NEAR.parse("0.05N") }
```

Comment:
Yeah. So the above call should fail in order to pass the test. Since this test is passing, that call should be failed. I think it is working correctly.